source/Ship.cpp

```diff
@@ -3258,6 +3263,24 @@ double Ship::JumpDriveFuel(double jumpDistance) const
 
 
 
+bool Ship::JumpDriveCheaper(const System *destination) const
+{
+	bool linked = currentSystem->Links().count(destination);
+	double hyperFuelNeeded = HyperdriveFuel();
+	double jumpFuelNeeded = JumpDriveFuel((linked || currentSystem->JumpRange()) ? 0. : currentSystem->Position().Distance(destination->Position()));
```

Comment:
```suggestion
	double jumpFuelNeeded = JumpDriveFuel((linked || currentSystem->JumpRange())
			? 0. : currentSystem->Position().Distance(destination->Position()));
```
Lines shound't be longer than 120 characters. The exact position of the wrap is somewhat arbitrary.